scripts/docs.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -e
+set -o pipefail
+set -u
+
+if [ -z `which jazzy` ]; then
+    echo "Installing jazzy…"
+    gem install jazzy
+    if [ -z `which jazzy` ]; then
+        echo "Unable to install jazzy."
+        exit 1
+    fi
+fi
+
+for module in MapboxNavigation MapboxCoreNavigation
+do
+    jazzy \
+        --config jazzy.yml \
+        --module "$module" \
+        --module-version 0.4.0 \
+        --root-url 'https://www.mapbox.com/navigation-sdk/ios/0.4.0/' \
+        --github-file-prefix 'https://github.com/mapbox/mapbox-navigation-ios/tree/master' \
```

Comment:
We should pin the file prefix to the current tag. That would necessitate pulling the module version out into a local variable. [This code](https://github.com/mapbox/mapbox-gl-native/blob/5e0018cdc3844994bf4afdc0a5e6ae90636c9967/platform/ios/scripts/document.sh#L18-L20) automatically detects the current release tag.